test/visualise_test.dart

```diff
@@ -0,0 +1,20 @@
+library many_points.test;
+
+import 'package:many_points/many_points.dart';
+
+void main() {
```

Comment:
This looks a bit more like an example than a test could we move it to the /example folder?